frontend/crash-recorded.css

```
@@ -230,6 +230,27 @@ summary:hover {
+  display: inline-flex;
+  align-items: center;
+  gap: 0.35rem;
+  padding: 0.25rem 0.5rem;
+  font-size: 0.75rem;
+  background: #262626;
+  border: 1px solid #383838;
+  border-radius: 4px;
+  color: #a3a3a3;
+  cursor: pointer;
+  transition: color 0.15s, background 0.15s;
+}
```

Stylelint (17.2.0)

[error] Expected empty line before rule (rule-empty-line-before)

(rule-empty-line-before)